compiler/rustc_span/src/symbol.rs

```diff
@@ -1262,6 +1262,7 @@ symbols! {
         self_in_typedefs,
         self_struct_ctor,
         semitransparent,
+        shadow_call_stack: "shadow-call-stack",
```

Comment:
This will probably want to be made into `shadow_call_stack` eventually – hyphens aren’t a valid identifier and most of the places this ends up used in the source code will likely be very ident-like.

But also see nikic’s comment about a test for `no_sanitize(shadow-call-stack)` – I suspect that will simply not work because of the hyphens. The argument should continue using hyphens though.

Reply:
Yup, the hyphens broke the no_sanitize attribute, so I've changed this to just `shadow_call_stack`.